features/measurements/api/src/main/java/org/opennms/netmgt/measurements/model/QueryResponse.java

```diff
@@ -136,6 +135,7 @@ public void setTimestamps(final ArrayList<Long> timestamps) {
         }
     }
 
+    @com.fasterxml.jackson.annotation.JsonSetter
```

Comment:
Small nit here and elsewhere in this file, import `JsonSetter` and then just use `@JsonSetter` as the annotation